web/pdf_viewer.js

```diff
@@ -2500,6 +2500,7 @@ class PDFViewer {
       this.#annotationEditorMode = mode;
       await this.#annotationEditorUIManager.updateMode(
         mode,
+        true,
```

Comment:
To make it a bit more clear what `true` represents here, given that there is no variable like `isFromKeyboard` that immediately makes the intention clear, we usually add an inline comment like this:

```
await this.#annotationEditorUIManager.updateMode(
  mode,
  editId,
  /* isFromUser = */ true,
  isFromKeyboard,
  ...
);
```

(also here with `editId` before the optional parameters)

Reply:
Done